transformations/add_hashtag/test.json

```diff
@@ -0,0 +1,49 @@
+{
+  "type": "hashtag_generation",
```

Comment:
this type should match the folder name - in your case: "add_hashtag"

Reply:
Completed all suggested changes.